[Calc Functions] Fully support space-separated lists

nex3 opened the pull request.

nex3 committed:
[Calc Functions] Fully support space-separated lists

Goodwine commented on accepted/calc-functions.md:
```diff
@@ -84,6 +83,10 @@ equivalent of `-10%` since percentages are resolved before calculations. To
 recommend users explicitly write `math.abs()` or `abs(#{})` instead.
 
+This also expands calculation parsing to allow constructs like `calc(1
+var(--plus-two))` which are valid CSS but weren't supported by the old
```
Nit: for clarity, please mention what `var(--plus-two)` means, something like _"where `--plus-two` has the value `+ 2`"_

Goodwine commented on accepted/calc-functions.md:
```diff
@@ -620,6 +625,36 @@ To evaluate a `SumExpresssion` or a `ProductExpression` as a calculation value:
+* Let `elements` be the results of evaluating each element as a calculation
+  value.
+
+* If `elements` has two adjacent elements that aren't unquoted strings, throw an
```
Would this accept `calc(a b)` and if so, is that OK?

nex3 replied:
Yes; I've updated the non-normative note to mention this.

nex3 committed:
Code review

nex3 committed:
Un-delete README symlink

nex3 committed:
Always preserve parentheses around unquoted strings

This ensures referential transparency, while any constants that Sass
supports will be evaluated as numbers anyway and so parens will be
omitted.
